code/controllers/configuration.dm

```diff
@@ -400,6 +401,9 @@ var/list/gamemode_cache = list()
 				if ("allow_ai")
 					config.allow_ai = 1
 
+				if ("allow_ai_drones")
+					config.allow_ai_drones = 1

```

Comment:
If it defaults to 0 (At variable declaration) and the config can only enable it, then there's no way to _dis_able it

Reply:
I set it the same way the allow_ai config is set by default, which is 1. I can change it if necessary, though it'll have to be marked for being enabled on the update.